src/components/GeneralInfo/SystemCard/SystemCard.test.js

```diff
@@ -449,29 +388,30 @@ describe('SystemCard', () => {
     });
   });
 
-  [
-    'hasHostName',
-    'hasDisplayName',
-    'hasAnsibleHostname',
-    'hasWorkspace',
-    'hasSAP',
-    'hasSystemPurpose',
-    'hasCPUs',
-    'hasSockets',
-    'hasCores',
-    'hasCPUFlags',
-    'hasRAM',
-  ].map((item, index) =>
-    it(`should not render ${item}`, () => {
+  const fieldsToTest = [
+    { prop: 'hasHostName', label: 'Host name' },
+    { prop: 'hasDisplayName', label: 'Display name' },
+    { prop: 'hasAnsibleHostname', label: 'Ansible hostname' },
+    { prop: 'hasWorkspace', label: 'Workspace' },
+    { prop: 'hasSystemPurpose', label: 'System purpose' },
+    { prop: 'hasCPUs', label: 'Number of CPUs' },
+    { prop: 'hasSockets', label: 'Sockets' },
+    { prop: 'hasCores', label: 'Cores per socket' },
+    { prop: 'hasCPUFlags', label: 'CPU flags' },
+    { prop: 'hasRAM', label: 'RAM' },
+  ];
+
+  fieldsToTest.forEach(({ prop, label }) => {
```

Comment:
nit-pick: generally consider using https://jestjs.io/docs/api#testeachtablename-fn-timeout as it is more stable and has better error reporting

Reply:
changed it to it.each